core/metrics/exporter.go

```diff
@@ -0,0 +1,222 @@
+			Help:      "How many Tenants exist",
+		})
+
+	metrics.agentsGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      agentsTotal,
+			Help:      "How many SHIELD Agents have been registered",
+		})
+
+	metrics.targetsGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      targetsTotal,
+			Help:      "How many Target Systems have been defined",
+		})
+
+	metrics.storesGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      storesTotal,
+			Help:      "How many Cloud Storage Systems have been defined",
+		})
+
+	metrics.jobsGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      jobsTotal,
+			Help:      "How many backup jobs have been defined",
+		})
+
+	metrics.tasksGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      tasksTotal,
+			Help:      "How many tasks have been created",
+		})
+
+	metrics.archivesGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      archivesTotal,
+			Help:      "How many Backup Archives have been generated",
+		})
+
+	metrics.storageUsedBytesGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      storageBytesTotal,
+			Help:      "How much storage has been used, in bytes.",
+		})
+
+	prometheus.MustRegister(metrics.tenantsGauge)
+	prometheus.MustRegister(metrics.agentsGauge)
+	prometheus.MustRegister(metrics.targetsGauge)
+	prometheus.MustRegister(metrics.storesGauge)
+	prometheus.MustRegister(metrics.jobsGauge)
+	prometheus.MustRegister(metrics.tasksGauge)
+	prometheus.MustRegister(metrics.archivesGauge)
+	prometheus.MustRegister(metrics.storageUsedBytesGauge)
+
+	metrics.tenantsGauge.Set(float64(config.TenantCount))
+	metrics.agentsGauge.Set(float64(config.AgentCount))
+	metrics.targetsGauge.Set(float64(config.TargetCount))
+	metrics.storesGauge.Set(float64(config.StoreCount))
+	metrics.jobsGauge.Set(float64(config.JobCount))
+	metrics.tasksGauge.Set(float64(config.TaskCount))
+	metrics.archivesGauge.Set(float64(config.ArchiveCount))
+	metrics.storageUsedBytesGauge.Set(float64(config.StorageUsedCount))
+
+	return metrics
+}
+
+func (m *Metrics) Inform(mbus *bus.Bus) {
+	m.bus = mbus
+}
+
+func (m *Metrics) ServeExporter() http.Handler {
+	return promhttp.Handler()
+}
+
+func (m *Metrics) CreateObjectCount(typeThing string, data interface{}) {
+	interfaceData := data.(map[string]interface{})
+	switch typeThing {
+	case "tenant":
+		m.tenantsGauge.Inc()
+	case "agent":
+		m.agentsGauge.Inc()
+	case "target":
+		m.targetsGauge.Inc()
+	case "store":
+		m.storesGauge.Inc()
+	case "job":
+		m.jobsGauge.Inc()
+	case "task":
+		m.tasksGauge.Inc()
+	case "archive":
+		m.archivesGauge.Inc()
+		m.storageUsedBytesGauge.Add(float64(interfaceData["size"].(int64)))
+	default:
+		log.Infof("Event type not recognized or not implemented yet.")
```

Comment:
``Metrics ignoring create event for object type `%s'`` where `%s` is the variable `typeThing`
```suggestion
		log.Debugf("Metrics ignoring create event for object type `%s'", typeThing)
```

Reply:
Also, change to a debug level log.